integration-cli/daemon.go

```diff
@@ -23,9 +23,7 @@ import (
 
 // Daemon represents a Docker daemon for the testing framework.
 type Daemon struct {
-	// Defaults to "daemon"
 	// Useful to set to --daemon or -d for checking backwards compatibility
```

Comment:
Looks like this comment is no longer relevant, but perhaps we should have a test for this? i.e. that `docker daemon` still works (if there's no test yet)